src/components/Layout.js

```diff
@@ -213,86 +226,91 @@ class Layout extends React.Component {
 
     return (
       <LayoutContainer ref="mainflex">
-        <Navigation headerColor={this.props.data.site.siteMetadata.headerColor}>
-          <NavigationBar>
-            <NavLogo
-              onClick={() => this.setState({ mobileMenuOpen: false })}
-              to={`/`}
-            >
-              <img src={logo} style={{ height: "48px", margin: "0px" }} />
-            </NavLogo>
-            <NavLinks>
-              <NavLink
-                to={`${localePrefix}#users`}
-              >
-                {i18n.users}
-              </NavLink>
-              <NavLink
-                to={`/en/developers/`}
-              >
-                {i18n.developers}
-              </NavLink>
-              <NavLink
-                to={`${localePrefix}#municipalities`}
+        <Header>
+          <MessageBar>
+            Starting from 3.4.2023, the use of Digitransit APIs will require registration and the registration is now open. <a href="/en/developers/api-registration">More information</a>
```

Comment:
Starting from 3.4.2023, use of Digitransit APIs will require registration. The registration is now open.